packages/fxa-email-service/src/api/send/mod.rs

```diff
@@ -77,29 +77,20 @@ impl FromDataSimple for Email {
 #[post("/send", format = "json", data = "<email>")]
 pub fn handler(
     email: AppResult<Email>,
-    bounces: State<DeliveryProblems<DbClient>>,
+    delivery_problems: State<DeliveryProblems<DbClient>>,
```

Comment:
Evidently this got missed when I did the big `bounce => delivery_problem` rename last year. Mopping it up now for consistency.